### PR TITLE
Bjs/260 custom home page hero for mobile

### LINKED ISSUE #260: Custom Home Page Hero for Mobile
Need Mobile/ Tablet Hero. Would like it to kick in as soon as corner of boxes crosses over "Watch Video" button crosses arrow.

When green triangle hits text above buttons, switch to mobile hero.

Watch and Weigh In Buttons overlap on mobile. Needs some padding.

### PR DESCRIPTION
**Pull request by smithWEBtek**
Resolves #260

# Why is this change necessary?
This is needed to provide mobile view of hero section.

# How does it address the issue?
It resizes and places the buttons upon the 'large' break point.
The green overlay covers the hero instead of being a triangle.
The rotating hero images stop at the break point.

# What side effects does it have?
None.

### CODE REVIEW

File: app/assets/stylesheets/components/styled-box.scss [hunk 1]
```diff
@@ -13,7 +13,7 @@
     background-position: center center;
     cursor: pointer;
     overflow: hidden;
-
+
```
Comment (mzagaja) [inline app/assets/stylesheets/components/styled-box.scss]: I think your text editor added extra white space. Would you consider deleting this from these blank lines?

File: app/assets/stylesheets/partials/_one_box.scss [hunk 2]
```diff
@@ -1,3 +1,3 @@
 .one-box {
-  height: 100%;
+  // height: 100%;
```
Comment (mzagaja) [inline app/assets/stylesheets/partials/_one_box.scss]: Why did you need to comment this out?

Comment (smithWEBtek) [reply app/assets/stylesheets/partials/_one_box.scss]: I must have been experimenting, should not be commented out, will fix.

**Comment by smithWEBtek** [comment]
@mzagaja the tests are all passing now, based on setting the window size.

The images are not actually cycling in the tests, I think because Capybara is not actually loading the page and waiting for the time loops like a user would experience.

But, the background image definitely does get set with the first hero_image created in the test. And, that image can only be set with the javascript working, which adds the "rotation-test" class that the tests currently check.

I feel like we shouldn't put more time into the ideal tests for image change, unless you know of a direct solution to mocking the actually user experience / time lapse to capture an actual image change.

**Comment by mzagaja** [comment]
@smithWEBtek I added a commit to make the tests actually look for the rotating image. We can discuss it in the morning, but otherwise this looks good for now. Let's get this merged.

**Comment by smithWEBtek** [comment]
@mzagaja beautiful!!
